tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/jvmmodel/SimpleTclGeneratorIntegrationTest.xtend

```diff
@@ -298,7 +298,7 @@ class SimpleTclGeneratorIntegrationTest extends AbstractTclGeneratorIntegrationT
 
 		// then
 		generatedCode.assertContains('''
-			reporter.enter(TestRunReporter.SemanticUnit.STEP, "Is <bar> visible?");
+			reporter.enter(TestRunReporter.SemanticUnit.STEP, "Is <bar> visible ?");
```

Comment:
This is only relevant for tests using the model to recreate this string. Using the node model this won't happen, but since the node model is not available here, the fallback is used.